tests/acl/test_acl.py

```diff
@@ -1268,6 +1268,8 @@ def post_setup_hook(self, dut, localhost, populate_vlan_arp_entries, tbinfo, con
             populate_vlan_arp_entries: A fixture to populate ARP/FDB tables for VLAN interfaces.
 
         """
+        if dut.get_facts()['asic_type'] == 'vs' and 't1' in tbinfo['topo']['name']:
```

Comment:
Why not use conditional mark to skip?

Reply:
In `tests_mark_conditions_acl.yaml`, there are 36 reboot acl testcases defined, so if skip in conditional mark, would need to add duplicate code for 36 times, maybe we can use this temporary, and add to conditional mark when the optimize done. But I also accept to add those duplicate code because it could skip cleaner